js/settings.js

```diff
@@ -36,6 +40,14 @@ function showSettings() {
         document.getElementById(jsonContainer).innerHTML = ""
         location.reload()
     }
+    document.getElementById('settings-cog').onclick = () => {
```

Comment:
@reesericci Why this part?? We don't need to this is manually right? Calling the `showSettings` function should suffice, it will automatically handle updating the JSON.

Reply:
so you can click the cog again to close

Reply:
I don't think the user would click the cog to close it because the settings menu already has a close button and when that menu is shown, the cog pretty much becomes invisible. It's better to just keep it that way rather than adding a bit more complexity?

Reply:
I think it might be better as I literally was helping someone set up this startpage and they literally closed out of the tab than clicking on the close button of the JSON window (lol). I think any way to make it easier to correctly close out of the JSON window is better than losing data. Besides, it's not that much more complexity. Although a better solution is to have it so if someone clicks outside the window, have it automatically close & save.